tally-up-backend/src/homeFunction.ts

```diff
@@ -0,0 +1,12 @@
+import type { APIGatewayProxyResultV2 } from 'aws-lambda';
+import fs from 'fs';
+
+export const handler = async (): Promise<APIGatewayProxyResultV2> => {
+  return {
+    statusCode: 200,
+    headers: {
+      'Content-Type': 'text/html',
+    },
+    body: fs.readFileSync('./index.html', 'utf-8'),
```

Comment:
wait.... is this the backend serving the index.html?  only the index though cause its still a static site too though right?

Reply:
1. ```bun run build```
2. This runs ```./tally-up-backend/scripts/build.ts```
3. This runs npm run build from ```./frontend```
4. Then copies everything from ```./frontend/dist``` to ```./backend/dist```
5. This then runs ```sam build```, which depends on template.yaml and outputs to ```.aws-sam/```

Reply:
There is a homeFunction serving index.html and a staticServeFunction serving main.js